pandas/tests/scalar/timestamp/test_unary_ops.py

```diff
@@ -298,7 +297,7 @@ def test_round_implementation_bounds(self):
         with pytest.raises(OverflowError, match=msg):
             Timestamp.max.ceil("s")
 
-    @pytest.mark.xfail(not IS64, reason="Failing on 32 bit build", strict=False)
+    @pytest.mark.xfail(reason="Failing on builds", strict=False)
```

Comment:
Could we check `raises=OverflowError` so this test is still somewhat useful? If not practical, I am still fine with this for now.

Reply:
Not sure if this is really practical, ceil works, the other two fail most of the time, but not sure if round fails every time